lib/utils.ts

```diff
@@ -59,15 +62,27 @@ export function expandTabs(line: string): string {
     });
 }
 
+function getRegexForTempdir(): RegExp {
+    const tmp = os.tmpdir();
+    return new RegExp(tmp.replaceAll('/', '\\/') + '\\/' + ce_temp_prefix + '[\\w\\d-.]*\\/');
+}
+
+/**
+ * Removes the root dir from the given filepath, so that it will match to the user's filenames used
+ *  note: will keep /app/ if instead of filepath something like '-I/tmp/path' is used
+ * @param filepath
+ * @returns
+ */
 export function maskRootdir(filepath: string): string {
```

Comment:
Can we not pass the "tmpdir" that's actually being used here? We know what it is: anything calling this will know the tmpDir _that particular compile_ is happening.

If that's the case; can this funcitonality not already be done in the same argument mapping code we have in `exec.ts` unconditionally? I get the feeling I'm missing something quite obvious here!

Reply:
this is always after the fact, and sometimes we have lost the information, but im happy to double check